configs/server/forseti_conf_server.yaml.in

```diff
@@ -523,15 +523,10 @@ notifier:
       cscc:
         enabled: false
         mode: api
-        # Alpha API
```

Comment:
Can you also please delete the alpha discover doc, as below?  There is no version number.
https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/google/cloud/forseti/common/gcp_api/discovery_documents/securitycenter.json